packages/ui/src/components/Modal/modal.tsx

```diff
@@ -65,7 +66,7 @@ const Modal: React.FC<ModalProps> = (props) => {
         <DefaultModal
             {...rest}
             prefixCls={MODAL_CLASS_PREFIX}
-            centered
+            centered={centered}
```

Comment:
why? all our modal are in center by default according to UI styleguides...

Don't add it to support some Frankenstein workarounds